Kronos-Shifts-Connector/README.md

```diff
@@ -74,7 +74,7 @@ Review and ensure AAD users, Teams, and Scheduling Groups are properly setup in
 Kronos requires a **Comment Text** value to be assigned to any comments or notes. This requires you to firstly configure each comment text before adding the chosen values in to the connector appSettings - this will be done when we deploy using the ARM template.
 
 The following settings you must configure are:
```

Comment:
You must configure the following settings:
or
The following settings must be configured:
? :)